README.md

```diff
@@ -14,6 +14,10 @@ C library.
 
 ## Installation ##
 
+### From a Named Release Tarball (Recommended) ###
+
+**NOTE:** These instructions are for installation from the _named_ `.tar.gz` tarballs on the [Releases](releases) page (e.g. `mod_maxminddb-*.tar.gz`).
```

Comment:
I'd use the full link here as a relative link will not work for all views of the file and this also gets distributed with the release tarball.

Also, please wrap the text at ~78 characters similar to the other text in the file.